Cluster creation fails with segmentation violation

```
$ ./eksctl create cluster
2018-06-05T13:31:55-07:00 [ℹ]  importing SSH public key "/Users/argu/.ssh/id_rsa.pub" as "EKS-wonderful-gopher-1528230715"
2018-06-05T13:31:56-07:00 [ℹ]  creating EKS cluster "wonderful-gopher-1528230715"
2018-06-05T13:31:56-07:00 [ℹ]  creating VPC stack "EKS-wonderful-gopher-1528230715-VPC"
2018-06-05T13:31:56-07:00 [ℹ]  creating ServiceRole stack "EKS-wonderful-gopher-1528230715-ServiceRole"
2018-06-05T13:32:16-07:00 [✔]  created ServiceRole stack "EKS-wonderful-gopher-1528230715-ServiceRole"
2018-06-05T13:33:16-07:00 [✔]  created VPC stack "EKS-wonderful-gopher-1528230715-VPC"
2018-06-05T13:33:16-07:00 [ℹ]  creating control plane "wonderful-gopher-1528230715"
2018-06-05T13:42:38-07:00 [✔]  created control plane "wonderful-gopher-1528230715"
2018-06-05T13:42:38-07:00 [ℹ]  creating DefaultNodeGroup stack "EKS-wonderful-gopher-1528230715-DefaultNodeGroup"
2018-06-05T13:46:18-07:00 [✔]  created DefaultNodeGroup stack "EKS-wonderful-gopher-1528230715-DefaultNodeGroup"
2018-06-05T13:46:18-07:00 [✔]  all EKS cluster "wonderful-gopher-1528230715" resources has been created
2018-06-05T13:46:18-07:00 [ℹ]  wrote "kubeconfig"
2018-06-05T13:46:23-07:00 [ℹ]  the cluster has 0 nodes
2018-06-05T13:46:23-07:00 [ℹ]  waiting for at least 2 nodes to become ready
2018-06-05T13:46:57-07:00 [ℹ]  the cluster has 2 nodes
2018-06-05T13:46:57-07:00 [ℹ]  node "ip-192-168-110-25.us-west-2.compute.internal" is ready
2018-06-05T13:46:57-07:00 [ℹ]  node "ip-192-168-168-114.us-west-2.compute.internal" is ready
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1ca60d7]

goroutine 1 [running]:
github.com/weaveworks/eksctl/pkg/utils.CheckKubectlVersion(0x0, 0x0)
	/go/src/github.com/weaveworks/eksctl/pkg/utils/kubectl.go:26 +0x167
github.com/weaveworks/eksctl/pkg/utils.CheckAllCommands(0x1ff4560, 0xa, 0x0, 0x0)
	/go/src/github.com/weaveworks/eksctl/pkg/utils/kubectl.go:49 +0x37
main.doCreateCluster(0xc4203286c0, 0xc420157cb0, 0xc420157c80)
	/go/src/github.com/weaveworks/eksctl/cmd/eksctl/create.go:158 +0x412
main.createClusterCmd.func1(0xc4201dbb80, 0x2a42b78, 0x0, 0x0)
	/go/src/github.com/weaveworks/eksctl/cmd/eksctl/create.go:55 +0x2a
github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra.(*Command).execute(0xc4201dbb80, 0x2a42b78, 0x0, 0x0, 0xc4201dbb80, 0x2a42b78)
	/go/src/github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2a1bc00, 0xc420157f68, 0x1ca8ba6, 0xc420157f78)
	/go/src/github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra.(*Command).Execute(0x2a1bc00, 0x1feffc3, 0x1)
	/go/src/github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/go/src/github.com/weaveworks/eksctl/cmd/eksctl/main.go:28 +0x2d
```

--- a/pkg/utils/kubectl.go
+++ b/pkg/utils/kubectl.go
@@ -23,7 +23,7 @@ func CheckKubectlVersion() error {
 	logger.Debug("kubectl: %q", kubectlPath)
 
 	clientVersion, _, err := kubectl.GetVersionInfo(ktl)
-	logger.Debug("clientVersion=%#v err=%q", clientVersion, err.Error())
+	logger.Debug("clientVersion=%#v err=%q", clientVersion, err)
 
 	version, err := semver.Parse(strings.TrimLeft(clientVersion, "v"))
 	if err != nil {

